src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Registry.UnitTest/Helper/QueryHelperTest.cs

```diff
@@ -17,6 +17,11 @@ public sealed class QueryHelperTest
         [DataRow(@"HKLM\\Test", true, @"HKLM", "Test")]
         [DataRow(@"HKLM\Test\\TestTest", true, @"HKLM\Test", "TestTest")]
         [DataRow(@"HKLM\Test\\\TestTest", true, @"HKLM\Test", @"\TestTest")]
+        [DataRow("HKLM/\"Software\"/", false, @"HKLM\Software\", "")]
+        [DataRow("HKLM/\"Software\"//test", true, @"HKLM\Software", "test")]
+        [DataRow("HKLM/\"Software\"//test/123", true, @"HKLM\Software", "test/123")]
+        [DataRow("HKLM/\"Software\"//test\\123", true, @"HKLM\Software", "test\\123")]
```

Comment:
Missing a test with the / in a key name. Currently only value names containing a / are tested.

Reply:
I've added some tests, have a look at them and let me know.